src/renderer/constants/strings.ts

```diff
@@ -0,0 +1,5 @@
+export const IMPORT_KEY_PLACEHOLDER = "Enter your unique mnemonic signing key";
```

Comment:
This needs a bit correction, in signing key it should say `Enter your mnemonic or private key` while in withrawal key, placeholder should be `Enter your withdrawal public key`